controllers/datadogagent/common_rbac.go

```diff
@@ -272,11 +272,20 @@ func isOwnerBasedOnLabels(dda *datadoghqv1alpha1.DatadogAgent, labels map[string
 	return isManagedByOperator && isPartOfDDA
 }
 
-func rbacNamesForDda(dda *datadoghqv1alpha1.DatadogAgent) []string {
+// rbacNamesForDda return the list of RBAC name that are used for cluster level RBAC
+func (r *Reconciler) rbacNamesForDda(dda *datadoghqv1alpha1.DatadogAgent) []string {
```

Comment:
To reduce coupling and maybe make some tests easier we could send the version info in the params instead of defining this function for `Reconciler`.